docs/whirlpool/package.json

```diff
@@ -15,8 +15,8 @@
     "prism-react-renderer": "^2.4.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "rehype-katex": "7",
-    "remark-math": "6"
+    "rehype-katex": "7.0.1",
```

Comment:
I would do "^7.0.1" instead of just "7.0.1". I believe it signifies `Any version that is backwards-compatible` with this version